database/migrations/20190304154730-create-user.js

```diff
@@ -0,0 +1,53 @@
+export default {
+  /**
+   * Run the migration
+   *
+   * @param {Sequelize.QueryInterface} queryInterface - The interface that communicates
+   * with the database.
+   * @param {Sequelize} sequelize - Sequelize object.
+   * @return {void}
+   */
+  up: (queryInterface, sequelize) => (
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: sequelize.INTEGER,
+      },
+      email: {
+        type: sequelize.STRING,
+      },
+      isConfirmed: {
+        type: sequelize.BOOLEAN,
+        defaultValue: false
+      },
+      password: {
+        type: sequelize.STRING,
+      },
+      createdAt: {
+        allowNull: true,
+        type: sequelize.DATE,
+        defaultValue: sequelize.literal('CURRENT_TIMESTAMP'),
+      },
+      updatedAt: {
+        allowNull: true,
+        type: sequelize.DATE,
+        defaultValue: sequelize.literal('CURRENT_TIMESTAMP'),
+        // onUpdate: sequelize.literal('CURRENT_TIMESTAMP'),
```

Comment:
#### Nitpick
- Why is this commented?

Reply:
I didn't comment this. Only thing I altered about the user is the association for the user model.